tools/scripts/scvi/scvi-config.yaml

```diff
@@ -0,0 +1,25 @@
+census:
+  organism: 
+    "mus_musculus"
+  obs_query:
+    'tissue_general == "heart" and is_primary_data == True'
```

Comment:
```suggestion
    'tissue_general == "heart" and is_primary_data == True and nnz >= 300'
```

Reply:
Actually probably need to deconstruct this to take advantage of `min_genes` below